### DISCUSSION TITLE
Working with human/demo/index.html

### DISCUSSION THREAD

**Discussion by saeidn95** [vladmandic/human#444] [Q&A]
Is there a procedure to work with  "human/demo/index.html"

I just dropped the index.html file into a browser and it does nothing. Do I need to set up a server or anything?

**Comment by vladmandic** [comment]
its covered in docs.
yes, you need http server. you can deploy your own, or use built-in on using `npm run dev`

**Reply by saeidn95** [reply]
I works! Thanks, I should be ok from now on to explore!
Thanks for making your work available publicly and being patient with me.
